### PR TITLE
TGMonkey 2.0

### PR DESCRIPTION
**Pull request by ghost**
Monkeys can now like you!

- [x] make it so monkeys can climb on your shoulder
- [ ] other friendly monkey things

:cl: More Robust Than You
add: Monkeys can now like you!
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
D'awwe.

**Comment by FantasticFwoosh** [comment]
Can monkeys leap onto your face via your shoulders to bite you now?

**Comment by MoreRobustThanYou** [comment]
@FantasticFwoosh good idea

**Comment by kevinz000** [comment]
before this is touched the shitcode needs to be cleaned out.
like them ignoring fucking stuns for one.

**Comment by FantasticFwoosh** [comment]
> before this is touched the shitcode needs to be cleaned out. 
🆗 

But where's your review?

**Comment by PKPenguin321** [comment]
Can poly ride your shoulder by himself yet? If we can do this we should be able to do that

**Comment by praisenarsie** [comment]
make sure bartender's monkey loves bartender and hates chef

**Comment by deathride58** [comment]
Make punpun capable of metagrudging/metafriending via persistent relationships

**Comment by MoreRobustThanYou** [comment]
@deathride58 that's actually an amazing idea

**Comment by kevinz000** [comment]
@MoreRobustThanYou i can't review code that isn't in close proximity to the lines you touched.

File: code/modules/mob/living/carbon/monkey/passive.dm [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	buckled = null
+	layer = initial(layer)
+	pixel_x = initial(pixel_x)
+	pixel_y = initial(pixel_y)
+
+/mob/living/carbon/monkey/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/reagent_containers/food))
+		var/obj/item/reagent_containers/food/food = W
+		if(food.foodtype & disliked_food)
+			visible_message("<span class='danger'>[src] looks at [W], but pushes it away.</span>", "<span class='danger'>We reject the gross-looking [W]!</span>")
```
Comment (kevinz000) [inline code/modules/mob/living/carbon/monkey/passive.dm]: wait so if a player is in the monkey they can't even choose to NOT eat it?

Comment (kevinz000) [reply code/modules/mob/living/carbon/monkey/passive.dm]: (this applies to the entire proc not just this)

**Comment by optimumtact** [comment]
stale ping for reopen etc etc, or make new pr bla bla you know what you did

**Pull request closed without merging**